src/containers/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from "react";
+import Navbar from "./Navbar";
+import ContentContainer from "../Dashboard/ContentContainer";
+
+class Dashboard extends Component {
+  render() {
+    return (
+      <div>
```

Comment:
If you are using div just because to make this in one tag you can use Fragment of react

Reply:
I have used Fragments instead of div tag